sidestep_engine/core/trainer_helpers.py

```
@@ -744,7 +744,48 @@ def resume_checkpoint(
+                for warn in ladder_result.warnings:
+                    yield TrainingUpdate(0, 0.0, f"[WARN] {warn}", kind="warn")
+            else:
+                # OFT and other non-LoRA adapters: skip ladder, use direct load_state_dict
+                try:
+                    decoder.load_state_dict(state_dict, strict=False)
+                    yield TrainingUpdate(
+                        0, 0.0,
+                        f"[OK] {adapter_type.upper()} adapter weights loaded directly (non-LoRA format)",
+                        kind="info",
+                    )
+                except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)